docs/GettingStarted/SpeedrunTricks.md

```diff
@@ -46,4 +49,21 @@ You'll know it worked because the "restart" will move back up to "resume" with n
 
 Doing the same but for "left" would set the 'dark' flag
 
-Further details can be found [here](ExpertCCRouteControl)
\ No newline at end of file
+Further details can be found [here](ExpertCCRouteControl)
+
+## Hurt Animation Cancel
+When damaged and Shadow is on the ground, pressing Y cancels the animation into the weapon drop animation.
```

Comment:
Does this require you to already have a weapon or is this always available?

Reply:
Requires a weapon